services/QuillLMS/app/models/classroom.rb

```diff
@@ -60,6 +56,11 @@ class Classroom < ApplicationRecord
 
   before_validation :set_code, if: proc {|c| c.code.blank?}
 
+  after_commit :hide_appropriate_classroom_units
+  after_commit :trigger_analytics_for_classroom_creation, on: :create
+
+  after_save :reset_teacher_activity_feed, :save_user_pack_sequence_items, if: :saved_change_to_visible?
```

Comment:
If we are moving these, let's use that opportunity to put them in [run order](http://guides.rubyonrails.org/active_record_callbacks.html#creating-an-object), `after_save` before `after_commit` (note, I believe I'm the one who put them out of order, so that's on me 🙃 ).

Reply:
Good call.  I tend to forget about the order.